install/installer/pkg/components/spicedb/data/bootstrap.yaml

```diff
@@ -0,0 +1,30 @@
+schema: |-
```

Comment:
This is an initial version, to have something. We'll be updating and refining this once we actually start using spicedb properly